### PR TITLE
Increase Xmx for HADOOP_CLIENT_OPTS for cdh5

### PR DESCRIPTION
**Pull request by homar**

### CODE REVIEW

File: archived/cdh5.15-hive/files/etc/hadoop/conf/hadoop-env.sh [hunk 1]
```diff
@@ -25,3 +25,6 @@ export HADOOP_HEAPSIZE=512
 # Extra Java runtime options.  Empty by default.
 export HADOOP_NAMENODE_OPTS="$HADOOP_NAMENODE_OPTS -Xmx512m"
 export YARN_OPTS="$YARN_OPTS -Xmx256m"
+
+# Necessary to prevent map reduce jobs triggered by hive queries from dying with OOM error
+export HADOOP_CLIENT_OPTS="$HADOOP_CLIENT_OPTS -Xmx512m"
```
Comment (findepi) [inline archived/cdh5.15-hive/files/etc/hadoop/conf/hadoop-env.sh:30]: You're changing the `archived` folder, you should consider it read-only.
Changing here won't have an affect on the build.

Also, the setting is copied from hive3.1-hive but we know that image isn't stable yet.
what is the effective Xmx in hdp3.1-hive image?

Comment (homar) [reply archived/cdh5.15-hive/files/etc/hadoop/conf/hadoop-env.sh]: For now I pushed a change where I increased HADOOP_HEAPSIZE to 768MB. 
It is not perfect and worse than I expected but it is the only setting that caused tests to pass each time.  Also it should be safe as it is the maximum allowed size so in most cases heap size should be far from it. 
I'd like to stress test this on Monday.

Comment (homar) [reply archived/cdh5.15-hive/files/etc/hadoop/conf/hadoop-env.sh]: it looks like this change helps both in trino https://github.com/trinodb/trino/pull/9580 and in sep https://github.com/starburstdata/starburst-enterprise/pull/3833

@findepi @losipiuk what do you think? can we merge it ?

File: testing/cdh5.15-hive/files/etc/hadoop/conf/hadoop-env.sh [hunk 2]
```diff
@@ -20,7 +20,7 @@
 # export HADOOP_MAPRED_HOME=/usr/lib/hadoop-mapreduce
 
 # The maximum amount of heap to use, in MB. Default is 1000.
-export HADOOP_HEAPSIZE=512
```
Comment (findepi) [inline testing/cdh5.15-hive/files/etc/hadoop/conf/hadoop-env.sh:23]: seems it was previously raised to 512 in 594826a by @ilfrin

Comment (homar) [reply testing/cdh5.15-hive/files/etc/hadoop/conf/hadoop-env.sh:23]: yep I saw that

**Pull request merged**